packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/shippingInformation.gql.js

```diff
@@ -34,11 +35,13 @@ export const SET_CUSTOMER_ADDRESS_ON_CART = gql`
                 id
                 ...ShippingInformationFragment
                 ...ShippingMethodsCheckoutFragment
+                ...PriceSummaryFragment
             }
         }
     }
     ${ShippingInformationFragment}
     ${ShippingMethodsCheckoutFragment}
+    ${PriceSummaryFragment}
```

Comment:
Do we actually need to include the `PriceSummaryFragment` in this query?

Shouldn't the price summary / order summary component already be requesting this data for itself?

Reply:
I don't think so. This is the same price summary component from the cart page and the `usePriceSummary` talon only re-fetches the price summary data when the cart id changes. If we add checks inside the talon to re-fetch on other variable changes, we will be breaking the existing code. I can still go ahead and make the change if you guys think that is the right way to handle this issue.

Reply:
I do think the easy and scalable approach is to let the components fetch what they want in this case shipping information to fetch order summary because it knows that order summary might change because of its operations and let apollo do its magic to let react know which components to re-render on new data, in this case, the order summary component since new data is available. This approach makes things simple and make order summary component/talon less bloated.

Reply:
Lol oops, GitHub compare messed me up - you didn't actually add the `PriceSummaryFragment` to the `GET_SHIPPING_INFORMATION` query 😝